docs/integrations/destinations/bigquery-migrations.md

```diff
@@ -1,5 +1,15 @@
 # BigQuery Migration Guide
 
+## Upgrading to 3.0.0
```

Comment:
we're maybe supposed to mention something about the new CDC hard/soft deletes option here? or maybe we just put that in the changelog?